resources/pgp-keys-map.list

```diff
@@ -14,13 +14,37 @@
 # limitations under the License.
 #
 
-# no signature for this version
-junit:junit:(,4.8)              =
+com.beust                       = 0x67497E9D680CE8E95BD6B8F85AD66315FC018797
+
+com.github.spotbugs:spotbugs-maven-plugin = 0x3B9CD44BEEDFBDFABA22717BEEA8F6DF3031CD02
+
+com.google.code.findbugs        = 0x7616EB882DAF57A11477AAF559A252FB1199D873
+
+com.google.errorprone:*:(,2.3.3]= 0xE77417AC194160A3FABD04969A259C7EE636C5ED
+com.google.errorprone:*:[2.3.4,)= 0x7615AD56144DF2376F49D98B1669C4BB543E0445
+
+com.pholser                     = 0x517B94F8D0A46317A28D8AB30DA8A5EC02D11EAD
 
+edu.berkeley.cs.jqf             = 0x07665A503E191C320F29C6C8E7C8ECE8FE1536DC
+
+info.picocli                    = 0x8756C4F765C9AC3CB6B85D62379CE192D401AB61
+
+junit:junit:(,4.8)              =
 # it is sub key of 0x58E79B6ABC762159DC0B1591164BD2247B936711
 # https://github.com/s4u/pgpverify-maven-plugin/issues/30
 junit:junit:[4.8,)              = 0xD4C89EA4AAF455FD88B22087EFE8086F9E93774E
 
+ognl                            = 0x8926173648953916A0A4F290F721C545D0CAA2E3
+
+org.antlr                       =
+
+org.apache.commons:commons-lang3:(,3.6] = 0xCD5464315F0B98C77E6E8ECD9DAADC1C9FCC82D0
+org.apache.commons:commons-lang3:[3.7]  = 0x2DB4F1EF0FA761ECC4EA935C86FDC7E2A11262CB
+org.apache.commons:commons-lang3:[3.8,) = 0xB6E73D84EA4FCC47166087253FAAD2CD5ECBB314
+org.apache.commons              = 0xB6E73D84EA4FCC47166087253FAAD2CD5ECBB314
```

Comment:
I think the we should put all this keys for groupId org.apache.commons, like for org.apache.maven.plugins

Reply:
Yeah, I guess that makes sense. Otherwise things become way too complicate to manage.